### PR TITLE
Update Drush

### PR DESCRIPTION
**Pull request by weitzman**
**Description:**
Routine update of Drush. This is done mostly to show that our Guthub Action is now working. See the comment below from @github-actions that shows what packages are updated in this PR.

**Jira:** (Skip unless you are MA staff)
DP-24125

**To Test:**
- [ ] Push this PR to a feature env. If it deploys successfully, Drush is working at Acquia and will reliably work when we push to Prod. Thats what we usually test when Drush updates.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: changelogs/DP-24125.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Changed:
+  - description: Improve cache hit rate for dynamic page cache
+    issue: DP-125
```
Comment (tom-fleming) [inline changelogs/DP-24125.yml]:
```suggestion
    issue: DP-24125
```

**Comment by weitzman** [comment]
The Behat failure looks unrelated to this PR. It would be important to fix that soon, outside of this PR.

**Comment by tom-fleming** [comment]
@weitzman This will be a handy change. Looks good to me. Just made one change to the changelog to point to the correct issue number.

**Review by tom-fleming — APPROVED** [review]

**Comment by weitzman** [comment]
We cannot replicate the failure locally. As it looks unrelated, I'm merging this. We can revert if needed.

**Pull request merged**